codalab/rest/workers.py

```diff
@@ -34,6 +34,7 @@ def checkin(worker_id):
         request.json.get("free_disk_bytes"),
         request.json["dependencies"],
         request.json.get("shared_file_system", False),
+        request.json.get("tag_exclusive", False),
```

Comment:
Hm, if we're going to have `tag_exclusive` after `shared_file_system` in some places, better to be consistent and put it after it everywhere, rather than having a mix.